### DISCUSSION TITLE
How to run dvc queue in a blocking manner?

### DISCUSSION THREAD

**Discussion by maxromanovsky** [treeverse/dvc#9521] [Help] [answered]
Could you suggest how to run a grid search in non-interactive way in the CI pipeline?

My idea was the following:
```sh
dvc exp run -S 'foo=5,10' -S 'bar=1,2' --queue
dvc exp run --run-all
```

However, [the docs say that](https://dvc.org/doc/command-reference/exp/run#--run-all):
> The --run-all and --jobs options will be deprecated in a future DVC release.

I can't reproduce this behaviour with `dvc queue start && dvc queue logs -f` (as suggested in the same docs) because `logs` subcommand requires task name as an argument.

**Comment by dberenbaum** [comment, marked as answer]
Please take a look at the discussion in https://github.com/iterative/dvc/issues/8658 and flag there if you have any concerns about the plans for this.
